ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml

```diff
@@ -4158,6 +4158,306 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build05
+  cron: 0 4 24 * *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+    timeout: 8h0m0s
+  extra_refs:
+  - base_ref: main
+    org: openshift-qe
+    repo: ocp-qe-perfscale-ci
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-perfscale-qe
+    ci-operator.openshift.io/variant: aws-ipsec-4.15-nightly-x86-loaded-upgrade-from-4.14
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-ipsec-4.15-nightly-x86-loaded-upgrade-from-4.14-loaded-upgrade-414to415-24nodes
+  spec:
```

Comment:
lets be sure to add the slack reporting here

Reply:
@paigerube14 Thanks. I added them. I found there are some other jobs missing the slack reporting, I added for them as well.

Reply:
@paigerube14 Others had provided their comments. If you are good with it now, please give a lgtm.

Reply:
I realized that you had to put this test in separate files because of the version type specified. We have a lot of files now. Maybe we need to look into updating this into the aws loaded-upgrade in the future